avocado/core/loader.py

```diff
@@ -717,7 +719,8 @@ def _make_avocado_tests(self, test_path, make_broken, subtests_filter,
                 else:
                     # Module does not have an avocado test class inside, and
                     # it's not executable. Not a Test.
-                    return make_broken(NotATest, test_path)
+                    return make_broken(NotATest,
```

Comment:
I thought about that (the default message per-class) but that would require another API in terms of getting the message and it did not looked important enough. We can always store these messages/mappings directly in loader plugin.

As for the `make_broken` sure, makes sense.